.golangci.yml

```diff
@@ -38,14 +38,11 @@ linters:
     - depguard
     - dupl
     - dupword
-    - errcheck
```

Comment:
Why disable this?

Reply:
Added back with assertion check-type-assertions=false